Components/Badge/badge.css

```diff
@@ -0,0 +1,68 @@
+body {
+  font-family: "Quicksand", sans-serif;
+  font-size: 20px;
+}
+
+.badge-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  border-radius: 1rem;
+  box-shadow: inset 0rem 0rem 0.5rem gray;
+  padding: 1rem;
+  margin: 0.5rem;
```

Comment:
suggestion - you can make utility classes to avoid code repetition and for your own flexiblity